code/datums/uplink_item.dm

```diff
@@ -400,7 +420,7 @@ var/list/uplink_items = list()
 
 /datum/uplink_item/dangerous/machinegun
 	name = "L6 Squad Automatic Weapon"
-	desc = "A fully-loaded Aussec Armoury belt-fed machine gun. This deadly weapon has a massive 50-round magazine of devastating 7.62x51mm ammunition."
+	desc = "A fully-loaded Aussec Armory belt-fed machine gun. This deadly weapon has a massive 50-round magazine of devastating 7.62x51mm ammunition."
```

Comment:
I was going to argue that Aussec Armoury is a proper noun for a specific Australian company but it looks like they just do security alarms and cctv, and the top google results for aussec armoury are...ss13 code...